Composer/packages/extension-client/src/hooks/usePublishApi.ts

```diff
@@ -58,6 +58,13 @@ export function usePublishApi() {
   function userShouldProvideTokens(): boolean {
     return window[ComposerGlobalName].userShouldProvideTokens();
   }
+  function getPrimaryToken(): boolean {
+    return window[ComposerGlobalName].getPrimaryToken();
+  }
+  function requireUserLogin(tenantId: string, options?: { graph: boolean }): boolean {
```

Comment:
Nit: require feels like it should throw if the user doesn't get logged in.  requestUserLogin would cause me to expect a value returned.

Reply:
yes we should commune on the best name for this method, which ... causes the login process to begin.